ReadMe.md

```diff
@@ -1,84 +1,105 @@
-#   Project Management - Api
-Project Management is application created under university science club - "Referencja"
+# Project Management - Api
```

Comment:
Czy nie powinien być Project Management - Client?

Reply:
Oj tak to jest mały błąd, prosze @RobertJaskolski  zwróć na to uwagę